src/Bridge/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -79,6 +79,7 @@ public function __construct(ManagerRegistry $managerRegistry, RequestStack $requ
     public function apply(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null, array $context = [])
     {
         if (!isset($context['filters'][$this->orderParameterName]) || !\is_array($context['filters'][$this->orderParameterName])) {
+            $context['filters'] = null;
```

Comment:
looks hackish no? Any other way we could fix that?

Reply:
My first though @soyuka was to remove the call to parent::apply() but it was breaking phpunit test suite. The issue to solve here is to call the grand parent function without calling the parent function.
To be more explicit, in this if we would need to call  `AbstractFilter::apply()` 
without calling `AbstractContextAwareFilter::apply()`, this is the way I found.

Reply:
Mhh yeah ok I think I got it. LGTM then thanks and nice catch btw!